File: Segundo Corte /Areas_y_Perimetros/Python/Square.py
class Square:
    def _init_(self, lado):
        print("Creating Square with side {l}".format(l=lado))
        self.lado = lado
    def Perimetro(self) :
        return 4*self.lado
    def area(self):
        return self.lado*self.lado
    
Square1 = Square(22)
Square2 = Square(4)
Square3 = Square(25)
        
print(Square1.Perimetro())
print(Square2.Perimetro())
print(Square3.Perimetro())

print(Square1.area())
print(Square2.area())
print(Square3.area())

File: Segundo Corte /Areas_y_Perimetros/Java/Square.java
package Square;

public class Square {
  
    public double lado;
    public Square(double lado){
        System.out.println("Creating Square with side " + lado);
        this.lado= lado;
    }
    public double Perimetro(){
    return 4*this.lado;
}
    public double Area(){
        return this.lado*this.lado;
    }
    public static void main(String[] args) {
        Square Square1 = new Square(12);
        Square Square2 = new Square(3);
        Square Square3 = new Square(24);
        
        System.out.println(Square1.Perimetro());
        System.out.println(Square2.Perimetro());
        System.out.println(Square3.Perimetro());
        
        System.out.println(Square1.Area());
        System.out.println(Square2.Area());
        System.out.println(Square3.Area());
    }
        
  }
